package.json

```diff
@@ -20,16 +20,15 @@
   },
   "dependencies": {
     "@crowdin/cli": "^3.7.10",
-    "@docusaurus/core": "^2.1.0",
-    "@docusaurus/preset-classic": "^2.1.0",
+    "@docusaurus/core": "^2.2.0",
+    "@docusaurus/preset-classic": "^2.2.0",
+    "@docusaurus/theme-mermaid": "^2.2.0",
     "@giscus/react": "^2.2.0",
     "@mdx-js/react": "^1.6.21",
     "axios": "^0.27.2",
     "clsx": "^1.1.1",
     "docusaurus-plugin-hotjar": "^0.0.2",
     "docusaurus-plugin-sass": "^0.2.1",
-    "mdx-mermaid": "^1.2.2",
-    "mermaid": "^8.14.0",
```

Comment:
docusaurus has support mermaid, so remove this.

Reply:
OK